### PR TITLE
Adding Go + removing redundancies

### PR DESCRIPTION
**Pull request by danield137**

### CODE REVIEW

**Comment by orspod** [comment]
@danield137 please make sure you add it to the TOC

File: data-explorer/kusto/api/index.md [hunk 1]
```diff
@@ -60,41 +61,7 @@ familiar database client tools (such as LINQPad, sqlcmd, Tableau, Excel, and Pow
 
 You can find more details about MS-TDS on [this page](tds/index.md).
 
-## .NET Framework Libraries
```
Comment (orspod) [inline data-explorer/kusto/api/index.md:63]: Why did you remove this entire section?

**Comment by orspod** [comment]
#sign-off

**Comment by tfosmark** [comment]
@orspod Please note that the new article (data-explorer/kusto/api/golang/kusto-golang-client-library.md) does not have the required metadata. This should be addressed asap. Thanks!
